### ISSUE TITLE
Search for id3v1 tag in file.

### ISSUE THREAD

**Issue by dogwatch** [ffalt/jamp3#96]
https://github.com/ffalt/jamp3/blob/ff77098bab90dfcaece63194b742ab3e5867b827/src/lib/mp3/mp3_reader.ts#L137

Is there a reason id3v1 is searched all over the file?

**Comment by ffalt** [comment]
yes and no. only the last 200 bytes of a current chunk are [checked](https://github.com/ffalt/jamp3/blob/ff77098bab90dfcaece63194b742ab3e5867b827/src/lib/mp3/mp3_reader.ts#L128). but i agree, if the stream size is available (it may be not), I can take that into account, too.
